module/enrichers.mjs

```diff
@@ -315,22 +315,39 @@ async function enrichSave(config, label, options) {
  * @param {string[]} config            Configuration data.
  * @param {string} [label]             Optional label to replace default text.
  *
- * @example Use a Heavy Crossbow:
+ * @example Use an item from a Name:
  * ```[[/item Heavy Crossbow]]```
  * becomes
  * ```html
  * <a class="roll-action" data-type="item">
  *   <i class="fa-solid fa-dice-d20"></i> Heavy Crossbow
  * </a>
  * ```
+ *
+ * @example Use an Item from a UUID:
+ * ```[[/item Actor.M4eX4Mu5IHCr3TMf.Item.amUUCouL69OK1GZU]]```
+ * becomes
+ * ```html
+ * <a class="roll-action" data-type="item">
+ *   <i class="fa-solid fa-dice-d20"></i> Bite
+ * </a>
+ * ```
 */
 
 async function enrichItem(config, label) {
   const givenItem = config.values.join(' ');
+  const itemUuidMatch = itemName.match(/^Actor\.\w{16}\.Item\.\w{16}$/);
```

Comment:
A uuid can also support having the name instead of an item's id.

Reply:
oops, I did leave in the wrong variable here, should have been `givenItem` instead of `itemName`  
But, that's not what you're talking about here, however, while a document link like `@Actor[]` will take a Name or Id, from what I can tell a UUID only takes Ids, something like `@UUID[Actor.Adult Black Dragon.Item.Bite]` does not enrich by default.  
While it would be nice to support these "pseudo-UUIDs" with names, since it would make manually creating these enrichers easier, it would offer little payoff in my opinion, since I think this method is really only useful for the Journal method mentioned earlier, and for Linked tokens.  
Unlinked tokens would be better served by the Name method or if I can get the Relative UUID method working, since the token's items could be different than their parent Actor's and the token could have effects active that the Actor wouldnt.

Reply:
Added support for using actor and item names in the UUID format, so now all of the following will work:
`[[/item Bite]]`
`[[/item Actor.Adult Black Dragon.Item.Bite]]`
`[[/item Actor.p26xCjCCTQm5fRN3.Item.amUUCouL69OK1GZU]]`
can even mix and match ids and names
`[[/item Actor.Adult Black Dragon.Item.amUUCouL69OK1GZU]]`
`[[/item Actor.p26xCjCCTQm5fRN3.Item.Bite]]`

![image](https://github.com/foundryvtt/dnd5e/assets/86370342/376f8123-759c-42bf-9300-6f00866c5313)